src/Cli/dotnet/commands/dotnet-list/dotnet-list-package/ListPackageReferencesCommandParser.cs

```diff
@@ -51,11 +54,25 @@ internal static class ListPackageReferencesCommandParser
         public static readonly Option InteractiveOption = new ForwardedOption<bool>("--interactive", CommonLocalizableStrings.CommandInteractiveOptionDescription)
             .ForwardAs("--interactive");
 
+        public static readonly Option VerbosityOption = new ForwardedOption<VerbosityOptions>(
+                new string[] { "-v", "--verbosity" },
```

Comment:
Elsewhere in the PR, you use CommonOptions.VerbosityOption. That can't be used here?

Reply:
Daniel had the same comment in the original PR. This command is special, it requires `--verbosity` to be passed in instead of the normal msbuild `-verbosity`.